pyproject.toml

```diff
@@ -13,6 +13,7 @@ line-length = 99
 target-version = ["py38"]
 
 [tool.coverage.report]
+fail_under = 100
```

Comment:
Wow :eyes:

Reply:
Yes, that's a high bar we'll have to maintain here, as it's not allowed to go down by team standard.